backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -3263,4 +3263,60 @@ databaseChangeLog:
         - sql:
             remarks: Populate new `emails` column with value from patient `email`
             sql: |
-              UPDATE person SET emails = ARRAY[email] WHERE email IS NOT NULL;
\ No newline at end of file
+              UPDATE person SET emails = ARRAY[email] WHERE email IS NOT NULL;
```

Comment:
This is part of another [PR](https://github.com/CDCgov/prime-simplereport/pull/3244)